app/townsquare/models.py

```diff
@@ -383,6 +383,29 @@ def __str__(self):
         return f"Favorite {self.activity.activity_type}:{self.activity_id} by {self.user}"
 
 
+class Pin(SuperModel):
+    """Model for each Pinned."""
```

Comment:
Pinned Post

Reply:
@owocki i will change the name of the class to Pinned Post